backend/v2/project_rewards/capped_quadratic.py

```diff
@@ -5,24 +5,24 @@
 
 from v2.allocations.schemas import AllocationWithUserUQScore
 from v2.core.types import Address
-from v2.project_rewards.schemas import ProjectFundingSummary
+from v2.project_rewards.schemas import ProjectFundingSummaryV1
 
 
-class CappedQuadriaticFunding(NamedTuple):
-    project_fundings: dict[Address, ProjectFundingSummary]
-    amounts_total: Decimal  # Sum of all allocation amounts for all projects
-    matched_total: Decimal  # Sum of all matched rewards for all projects
+class CappedQuadraticFunding(NamedTuple):
+    project_fundings: dict[Address, ProjectFundingSummaryV1]
+    allocations_total_for_all_projects: Decimal  # Sum of all allocation amounts for all projects
```

Comment:
are the comments useful now after we changed the variable name and it's sef-describing?